frontend/appflowy_flutter/assets/translations/hi.json

```diff
@@ -0,0 +1,441 @@
+{
+  "appName": "ऐप फ्लोवी",
+  "defaultUsername": "में",
+  "welcomeText": "स्वागत है @:appName",
+  "githubStarText": "गिटहब पैर स्टार दे",
+  "subscribeNewsletterText": "हमारे न्यूज़ की जानकारी पाए",
+  "letsGoButtonText": "शुरू करे",
+  "title": "टाइटल",
+  "signUp": {
+    "buttonText": "साईन करे",
+    "title": "साईन करे @:appName",
+    "getStartedText": "शुरू करे",
+    "emptyPasswordError": "पासवर्ड खाली नहीं रख सकते.",
```

Comment:
I think it's inappropriate to end phrases with full stop in Hindi, you can remove them.